index.html

```diff
@@ -2,7 +2,8 @@
 layout: landing
 title: Spine Event Engine
 ---
-
+<!--Todo: add a reworked product description. Current proposal to be discussed with PR and marketing team: "Apply reactive Domain-Driven Design quicker, with less code transforming EventStorming into a domain model code.
```

Comment:
Please remove the comment and have an issue.

Reply:
LGTM
Removed from the source code. 
Converted to documentation issue #8
[https://github.com/SpineEventEngine/documentation/issues/8]